src/libraries/System.Private.CoreLib/src/Resources/Strings.resx

```diff
@@ -3344,6 +3344,9 @@
   <data name="RFLCT_Targ_ITargMismatch" xml:space="preserve">
     <value>Object does not match target type.</value>
   </data>
+  <data name="RFLCT_Targ_ITargMismatch_WithType" xml:space="preserve">
```

Comment:
Is the string above now dead?

Reply:
It's still used by  runtime code in `interoputil.cpp`.  @steveharter suggested leaving the original string for use within the runtime code and add a new string for managed - https://github.com/dotnet/runtime/issues/97022#issuecomment-1896762339